linode/manager
# Fix Storybook Dependency Issues

## Description

- Updates Sentry and fixes major dependency issues encountered by @tiffwong and myself
  - Resolves https://github.com/storybookjs/storybook/issues/16837 for Cloud Manager
- Should help us un-block our dependency upgrade tickets

## How to test

- Run `yarn clean`
- Run `yarn`
- Run `yarn up` and check for issues and Typescript errors (especially lookout for react-router related errors)
- Run `yarn storybook` and check Storybook functionality
